Riot/Managers/Theme/ThemeService.m

```diff
@@ -146,6 +146,12 @@ - (void)updateAppearance
 {
     [UIScrollView appearance].indicatorStyle = self.theme.scrollBarStyle;
     
+    // Remove the extra height added to section headers in iOS 15
+    if (@available(iOS 15.0, *))
```

Comment:
Below `if #available(iOS 15.0, *)` is used, is this another correct way of doing it?